### ISSUE TITLE
failed packaging

### ISSUE THREAD

**Issue by SMaiLubis** [artoolkitx/arunityx#20]
i use windows 7
i already run build.sh and get this result:
![build](https://user-images.githubusercontent.com/46646098/58134885-325be300-7c63-11e9-8039-7efc9f110a9e.jpg)

and then i try to packaging i get this error:
![packaging](https://user-images.githubusercontent.com/46646098/58134931-6f27da00-7c63-11e9-8aae-d5bcc9642029.jpg)

can you help me? i already follow the instruction in https://github.com/artoolkitx/arunityx/blob/master/Quick%20start.md

**Comment by louisch** [comment]
The `packaging.sh` file right now seems to assume macOS. That particular line seems to rename lines in a particular file (`$ARUNITYX_HOME/Source/Package/Assets/artoolkitX-Unity/Scripts/Editor/ARToolKitMenuEditor.cs`) to the correct version using `sed`, but I don't know whether the version of `sed` on macOS is different, because that line seems to be using the wrong syntax. I fixed it by deleting the `''` in `sed -Ei ''` (the first set of single-quotes), and also for some reason `packaging.sh` seemed to assume some paths were lowercase when they were mixed for me, so I had to change those lines too. Assuming your `sed` isn't too different to mine that hopefully will work.

Additionally, the line directly below that one (line 56) assumes that Unity is installed in `/Applications`. As in, that you're running macOS and you've installed Unity as a application. You would have to change that to wherever you've installed Unity on your Windows computer.
